lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -29,25 +29,25 @@
 options:
   source_region:
     description:
-      - The source region the AMI should be copied from.
+      - the source region that AMI should be copied from
```

Comment:
why lowercase "The" and remove 2nd the? original needed correction but it should look like this:

```The source region that the AMI (image) should be copied from.```